uint8 servo_1
uint8 servo_2
uint8 servo_3
uint8 servo_4
